map_model/src/map.rs

```diff
@@ -182,7 +183,7 @@ impl Map {
         &self.roads
     }
 
-    pub fn all_lanes(&self) -> &Vec<Lane> {
+    pub fn all_lanes(&self) -> &BTreeMap<LaneID, Lane> {
```

Comment:
This caused kind of annoying amount of callers to need a `.values()` to iterate over the result. Maybe it's worth returning the iterator trait object or whatever and doing `self.lanes.values()` here? But meh.

Reply:
Yeah `-> impl Iterator<Item=LaneID>` seems neater and better encapsulated, especially if we switch data structures again. Though, I'm fine with it as you have it.